src/RiakClient/Config/IRiakClusterConfiguration.cs

```diff
@@ -53,6 +53,11 @@ public interface IRiakClusterConfiguration
         bool UseTtbEncoding { get; set; }
 
         /// <summary>
+        /// Set to <b>true</b> to disable exceptions thrown to prevent expensive operations.
+        /// </summary>
+        bool DisableListExceptions { get; set; }
```

Comment:
What do you think about renaming this `allowListing` or `allowListingOperations` like the other clients?  "DisableListExceptions" keeps throwing me off when seeing the negative and a boolean together.